### PR TITLE
Task/login form

### PR DESCRIPTION
**Pull request by Ral15**
What's this PR do?
This PR includes the full implementation of the login logic and making the login view the main one.

Where should the reviewer start?
express-app/controllers/user.js

How should this be manually tested?
***There are no tests yet for this PR. 

Any background context you want to provide?
Do not MERGE, this only for you guys to look at the documentation in the code and help me improve doing all this stuff.

This template was adapted ~stolen~ from Quickleft/Sprint.ly

### CODE REVIEW

File: express-app/controllers/user.js [hunk 1]
```diff
@@ -1,33 +1,83 @@
...
+   * 
+   * @event
+   * @param {object} data - data from form
+   * @param {object} request - request object 
+   * @param {object} response - response object.
+   */  
+  requestUser: function(data, request, response) {
+    req.post(
+      //url to post
+      'http://127.0.0.1:8000/perfiles-usuario/api-login/',
```
Comment (erikiado) [inline express-app/controllers/user.js]: I think this should be part of route/urls.json, and it should be used here

File: express-app/views/dashboard.hbs [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
           </div>
+          <h1>Hola {{user.username}}</h1>
+          <button class="btn btn-primary" id="estudios-btn">Estudios</button>
         </div>
       </div>
+    </div>
+    <!--Import jQuery before materialize.js-->
       <!--Import jQuery before materialize.js-->
-      <script type="text/javascript" src="/js/materialize.min.js"></script>
+      <script src="https://code.jquery.com/jquery.js"></script>
```
Comment (erikiado) [inline express-app/views/dashboard.hbs]: We need to download every package in order to serve it locally for the offline app

File: express-app/views/index.hbs [hunk 3]
```diff
@@ -3,7 +3,7 @@
   <head>
     {{> head}}
   </head>
-  <body>  
+  <body class="login-bg">
```
Comment (erikiado) [inline express-app/views/index.hbs:6]: why is this class on the whole body?

**Review by erikiado — CHANGES_REQUESTED** [review]

File: express-app/views/index.hbs (hunk 3 above)
Comment (Ral15) [reply express-app/views/index.hbs:6]: If you see in the template they use it that way. Look in _login.html_  line 18. @erikiado

**Pull request merged**